node_modules/oae-core/resetpassword/bundles/af_ZA.properties

```diff
@@ -0,0 +1,8 @@
+#X-Generator: crowdin.com
```

Comment:
It looks like these translation files aren't actually related to the feature under development, so they should be taken out.